src/gui/src/displayControls.cpp

```diff
@@ -999,13 +999,15 @@ void DisplayControls::techInit()
         color = colors[metal++];
       } else {
         // pick a random color as we exceeded the built-in palette size
+        // coverity[DC.WEAK_CRYPTO]
```

Comment:
What are the comments for?  Is this an exclusion?  If so you can just mark it in Coverity as Action: ignored.

Reply:
Yes, I'll revert this then